### PR TITLE
feat: add `PersonsDeactivationJob`

### PR DESCRIPTION
**Pull request by novohatnya**

### CODE REVIEW

File: __schema__/index.graphql [hunk 1]
```diff
@@ -258,6 +259,25 @@ type Query {
   "Reads a single `  PersonsAuthResetJob` using its globally unique ID."
```
Comment (ephor) [inline __schema__/index.graphql]:
```suggestion
  "Reads a single `PersonsAuthResetJob` using its globally unique ID."
```

**Comment by burlachenkovv** [comment]
Don't we need to show the `reason` field in the `Job` or `Task` info?

**Comment by kiurchv** [comment]
@burlachenkovv It makes sense to add them to job metadata, I'll find out how we can do it

**Comment by PavelVesnin** [comment]
@kiurchv metadata is used by client and contains 3rd party data. `Reason` is not a part of client's data, so it's shouldn't be in `meta`

**Pull request merged**